### PR TITLE
fix typos in README

### PR DESCRIPTION
**Pull request by Saibo-creator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,7 +477,7 @@ def calculator_call(lm):
 @guidance
 def calculator(lm):
     expression = lm['tool_args']
-    # You typically don't want to run eval directly for save reasons
+    # You typically don't want to run eval directly for safe reasons
```
Comment (riedgar-ms) [inline README.md:480]: I think that 'security' might be a better word here

**Review by riedgar-ms — COMMENTED** [review]
Thanks for working on these!

**Comment by Saibo-creator** [comment]
> Thanks for working on these!

It's nothing, I was just  trying to understand how Guidance works :') 

(P.S. updated the word safe for security)

**Review by riedgar-ms — APPROVED** [review]
LGTM

**Pull request merged**